MELA/interface/TVar.hh

```diff
@@ -26,7 +26,7 @@ typedef std::vector<SimpleParticle_t> SimpleParticleCollection_t;
 
 
 namespace TVar{
-  enum{
+  enum KAssociated{
```

Comment:
These are internal flags, so I don't think they need an enum class structure. Naming should also not start with `K`.